packages/action-group/src/ActionGroup.ts

```diff
@@ -295,10 +301,14 @@ export class ActionGroup extends SizedMixin(SpectrumElement, {
                     this.buttons.forEach((option) => {
                         option.setAttribute('role', 'button');
                     });
-                    this.removeAttribute('role');
                     break;
                 }
         }
+
+        // When no other role is defined, use role="toolbar", which is appropriate with roving tabindex.
+        if (!this.hasAttribute('role')) {
+            this.setAttribute('role', 'toolbar');
```

Comment:
If we put this at the top of the `default` block above, would that allow logic free application of the role?

Reply:
See the test that was added: https://github.com/adobe/spectrum-web-components/pull/3227/files/7ea146535ccb7cfdf441cdead65d1c1434b6a246#diff-c7346234f129e6f42044c0dab13aa3c69a97010a58c078bb4bd10466bce543f5R971

Reply:
Cool. So `role="group"` should only appear as a descendant of a `role="toolbar"` parent? This is super useful info for a consumer, and paired with the demo below would make a great "Accessibility" section at the end of the docs for this pattern. I'm happy to follow up with that after the fact, if I could steal you to review the PR when it comes. Or, if you were interested in including something yourself, it would be a powerful tool for the tool heavy workflows our consumers are working with!

Reply:
I can add the example below to the docs, which should help clarify when it may be desirable to override the default `role="toolbar"`.